pycontrails/datalib/ecmwf/model_level_hres.py

```diff
@@ -0,0 +1,511 @@
+"""Model-level HRES data access from the ECMWF operational archive.
+
+This module supports
+
+- Retrieving model-level HRES data by submitting MARS requests through the ECMWF API.
+- Processing retrieved GRIB files to produce netCDF files on target pressure levels.
+- Local caching of processed netCDF files.
+- Opening processed and cached files as a :class:`pycontrails.MetDataset` object.
+
+This module requires the following additional dependency:
+
+- `metview (binaries and python bindings) <https://metview.readthedocs.io/en/latest/python.html>`_
+"""
+
+from __future__ import annotations
+
+import contextlib
+import hashlib
+import logging
+import warnings
+from datetime import datetime, timedelta
+from typing import Any
+
+LOG = logging.getLogger(__name__)
+
+import pandas as pd
+import xarray as xr
+from overrides import overrides
+
+import pycontrails
+from pycontrails.core import cache, datalib, met_var
+from pycontrails.core.met import MetDataset, MetVariable
+from pycontrails.datalib.ecmwf import variables as ecmwf_var
+from pycontrails.datalib.ecmwf.common import ECMWFAPI
+from pycontrails.datalib.ecmwf.model_levels import pressure_levels_at_model_levels
+from pycontrails.utils import dependencies, temp
+from pycontrails.utils.types import DatetimeLike
+
+MODEL_LEVEL_VARIABLES = [
```

Comment:
Are these unique to Model Levels?

Reply:
0c24a63e661e903787c7b0c6b94bac1e9a667e19